src/datatypes/colorizer.md

```diff
@@ -2,18 +2,29 @@
 
 A colorizer specifies a mapping between values and pixels/objects of an output image.
 Different variants of colorizers perform different kinds of mapping.
+In general, there are two families of colorizers: **_gradient_** and **_palette_**.
+_Gradients_ are used to interpolate a continuous spectrum of colors between explicitly stated tuples (`breakpoints`) of a value and a color.
+A `palette` colorizer on the other hand, is used to generate a discrete set of colors, each mapped to a specific value.
 
-Usually, there are two miscellaneous fields in each colorizer, namely `noDataColor` and `defaultColor`.
+There are three miscellaneous fields in both of the gradient colorizers, namely `noDataColor`, `overColor` and `underColor`.
 The field `noDataColor` is used for all missing, `NaN` or no data values.
-The `defaultColor` is used for all overflowing values, for instance, if there are breakpoints defined from `0` to `10`, but a value of `-5` or `11` is mapped to a color.
+The fields `overColor` and `underColor` are used for all overflowing values.
+For instance, if there are breakpoints defined from `0` to `10`, but a value of `-5` or `11` is mapped to a color, the respective field will be chosen instead.
+This way, you can specifically highlight values that lie outside of a given range.
 
+For a `palette` colorizer, there are no `overColor` and `underColor` fields.
+Instead, a `noDataColor` is used as a fallback and `defaultColor` is mapped to any value not explicitly declared in the palette's definition.
```

Comment:
"Instead" makes no sense with respect to no data, since this is used for no data values and not as a fallback.

The default color is the fallback for all values that do not match any specified value.

Reply:
[1ee490a](https://github.com/geo-engine/docs/commit/1ee490a4e4b6705aec3bf9ce98ba5996159d2d97#diff-5bf7663539adc56559802f7881e6c9cc03912949d6a679e8ebd4227da9e3be3eR16)